README.md

```diff
@@ -45,6 +45,7 @@ custom:
     stages:
       - '[name of the stage to apply log forwarding]'
       - '[another stage name to filter]'
+    disabledLambdaPermission: false # whether to disable creation of the AWS::Lambda::Permission for the destinationARN (when policy size limits are a concern)
```

Comment:
It should be obvious, but could you add on a blurb stating that folks are expected to create this lambda permission on the destination lambda if they disable it in the plugin?